lte/gateway/c/core/oai/tasks/amf/amf_authentication.cpp

```diff
@@ -652,6 +663,11 @@ int amf_proc_authentication_failure(
 
   ue_mm_context = amf_ue_context_exists_amf_ue_ngap_id(ue_id);
   if (!ue_mm_context) {
+    OAILOG_WARNING(
+        LOG_NAS_AMF,
+        "Sending Auth Reject as UE is not authorized due to illegal UE "
```

Comment:
illegal UE? Simply saying "Sending Auth Reject as UE MM Context is not found" be sufficient.

Reply:
Changed the log message as suggested